.claude/docs/mega-prompt-endurecimiento.md

```
@@ -0,0 +1,290 @@
+1. <acción exacta, ejecutable, sin ambigüedad>
+
+## Bloqueado / pendiente de decisión
+- <qué, por qué, quién decide>
+
+## Cómo verificar que todo sigue sano
+```bash
+cd RAIZ && CMD_VALIDATE && python scripts/check_doc_links.py
+```
+
+## Línea base de métricas
+| Métrica | Antes | Después |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)